src/base-element.js

```diff
@@ -503,8 +530,8 @@ export class BaseElement {
    * @param {!Element|!Array<!Element>} elements
    * @protected
    */
-  schedulePause(elements) {
-    this.resources_.schedulePause(this.element, elements);
+  scheduleUnload(elements) {
```

Comment:
`amp-video` only implements a `#pauseCallback`, so it won't actually unload. But things like `amp-iframes` should definitely be unloaded when the user swipes.